Rlog/View/WorkSpace/WorkSpaceListView.swift

```diff
@@ -27,6 +29,11 @@ struct WorkSpaceListView: View {
                         .fontWeight(.bold)
                 }
                 ToolbarItem(placement: .navigationBarTrailing) {
+                    // MARK: 네비게이션으로 변경 필요
+                    //                    NavigationLink(destination: WorkSpaceCreateView(isActive: $isActive), isActive: $isActive) {
+                    //                        Image(systemName: "plus")
+                    //                            .fontWeight(.bold)
+                    //                    }
                     Button(action: { print("button pressed") }){
                         Image(systemName: "plus")
                             .fontWeight(.bold)
```

Comment:
```suggestion
                                        NavigationLink(destination: WorkSpaceCreateView(isActive: $isActive), isActive: $isActive) {
                                            Image(systemName: "plus")
                                                .fontWeight(.bold)

```
요런식으로 바꿔주시면 적용됩니다.
이부분은 혹시 꼬일까봐 주석으로 작성했습니다.